lib/controllers/create.js

```diff
@@ -13,7 +13,9 @@ newArticleQueue.process(job => {
 	if (!job.data.id || !job.data.title) {
 		return Promise.resolve();
 	}
-	
+
+	const webUrl = process.env.APP_URL + `content/${job.data.id}`;
```

Comment:
```suggestion
  const path = require('path');
	const webUrl = path.join(process.env.APP_URL, `/content/${job.data.id}`;
```

As the APP url is hidden within a environment variable here I think it would be nice use `path.join` to make sure we have the correct `/` in place.

for instance it would stop `https://ftalphaville.ft.com/longroom/` + `/content/1234'` from becoming  `https://ftalphaville.ft.com/longroom//content/1234`

Reply:
Great! I'll update and redeploy 👍

Reply:
Thanks 😄 sorry for being slow